File: Pyhon/1181.py
tam = 12
l = int(input())
t = input()
m = [[0.0 for i in range(tam)]for j in range(tam)]

cont = 0
media = 0
soma = 0

for i in range(tam):
    for j in range(tam):
        m[i][j] = float(input())

if (t == 'S'):
   for j in range(tam):
      soma+= m[l][j]
   print(soma)

if(t == 'M'):
   for j in range(tam):
      media += m[l][j]
   media = media/tam
   print(media)

File: C/1181.c
#include <stdio.h>
#include <stdlib.h>

int main(){
	const int tam = 12;
	float matriz[tam][tam],media,soma=0;
	int i,j,l;
	char t;

	scanf("%d",&l);
	scanf(" %c",&t);

	for (i = 0; i < tam; ++i)
	{
		for (j = 0; j < tam; ++j)
		{
			scanf("%f",&matriz[i][j]);
		}
	}
    for(i=0;i<tam;i++){
        for ( j = 0; j < tam; ++j)
        {
            if(i == l){
            soma = soma + matriz[i][j];
            }
        }
    }
    if(t=='S')
        printf("%.1f\n",soma);

	if (t == 'M')
	{
		media = soma/tam;
		printf("%.1f\n",media );
	}

	return 0;
}
